lib/byron/src/Cardano/Wallet/Byron/Transaction/Size.hs

```diff
@@ -194,20 +192,20 @@ instance MinSizeOf Address ('Testnet pm) IcarusKey where minSizeOf = 47
 --         = CBOR-LIST-LEN (2)    --     1 byte
 --         | CBOR-TAG (24)        --     2 bytes
 --         | CBOR-BYTES (33)      --     2 bytes
---         | 66OCTET              --    66 bytes #------*
+--         | 62-66OCTET           -- 62-66 bytes #------*
 --         | U32                  --   1-5 bytes        |
 --                                                      |
 --         *--------------------------------------------*
 --         |
 --         v
---         66OCTET
+--         62-66OCTET
 --             = CBOR-LIST-LEN (3)    --     1 byte
 --             | CBOR-BYTES (28)      --     2 bytes
 --             | 28OCTET              --    28 bytes
---             | ATTRIBUTES (34)      --    34 bytes
+--             | ATTRIBUTES (34)      -- 30-34 bytes
 --             | U8                   --     1 bytes
-instance MaxSizeOf Address 'Mainnet ByronKey where maxSizeOf = 77
-instance MinSizeOf Address 'Mainnet ByronKey where minSizeOf = 73
+instance MaxSizeOf Address 'Mainnet ByronKey where maxSizeOf = 76
```

Comment:
These `maxSizeOf` values doesn't include the list len byte. I assume the code is correct and comment ambiguous perhaps.

Reply:
Yeah, the comment is a bit ambiguous. I tried to improve things compare to what it was back on the http-bridge, but it's still far from perfect. I guess the next update is going to be Shelley with a non-ambiguous CDDL spec.